apps/dashboard/test/apps/ood_app_test.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class OodAppTest < ActiveSupport::TestCase
+  test 'version should return null when Configuration.hide_app_version? is true' do
+    Configuration.stubs(:hide_app_version?).returns(true)
+    under_test = OodApp.new RouterMock.new('test/fixtures/apps/ood_app/with_version')
```

Comment:
Do we need the directory `ood_app` or can we just have `test/fixtures/apps/with_version`?

Reply:
Will update

Reply:
fixed